base/strings/util.jl

```diff
@@ -446,7 +448,7 @@ end
 # check for pure ASCII-ness
 
 function ascii(s::String)
-    for (i, b) in enumerate(s.data)
+    for (i, b) in enumerate(bytes(s))
```

Comment:
It's an iterator over bytes that I was trying out. I guess it's not really necessary.